.github/workflows/ci.yml

```
@@ -0,0 +1,85 @@
+        run: ./gradlew spotlessCheck
+
+  get-keyword:
+    runs-on: ubuntu-latest
+    outputs:
+      level: ${{ steps.get-keyword.outputs.level }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v7
+      - name: get keyword
+        id: get-keyword
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2076:warning:5:46: Remove quotes from right-hand side of =~ to match as a regex rather than literally

(shellcheck)